doc/TODO.md

```diff
@@ -1,28 +1,33 @@
 DPVS TODO list
 ==============
 
-* [x] IPv6 Support.
-* [x] Documents update.
+* [x] IPv6 Support
+* [x] Documents update
 * [ ] NIC without Flow-Director (FDIR)
-  - [x] Packet redirect to workers.
-  - [ ]RSS pre-calcuating.
-* [ ] Merge lastest DPDK stable
+  - [x] Packet redirect to workers
+  - [ ] RSS pre-calcuating
+* [x] Merge DPDK stable 18.11
+* [ ] Merge DPDK stable 20.11
+* [ ] Service whitelist ACL
```

Comment:
I rmb this is done?

Reply:
The feature is not merged for this version.